dvc/cli.py

```diff
@@ -117,6 +117,37 @@ def parse_args(self, args=None, namespace=None):
             self.error(msg % " ".join(argv), getattr(args, "func", None))
         return args
 
+    def parse_known_args(self, args=None, namespace=None):
+        self.add_common_args()
```

Comment:
Btw, this might mess with `shtab`, as now `-v/-q/etc` are added dynamically when command is being parsed. formatter-based proposed potential solution from https://github.com/iterative/dvc/pull/4681#discussion_r502950934 doesn't seem to suffer from that.